experiments/AtmosLES/dycoms.jl

```diff
@@ -333,6 +333,15 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
         solver_method = LSRK144NiegemannDiehlBusch,
     )
 
+    #= Multirate:
+    ode_solver = ClimateMachine.MultirateSolverType(
```

Comment:
Could we set this up so only the `solver_type` keyword needs to be changed, and then the `Courant_number` keyword depends on the solver type? Commented switches in `master` are perhaps not the best approach going forward.